### PR TITLE
BRO 21 more 11 NOV

### PR DESCRIPTION
**Pull request by Simisays**

### CODE REVIEW

File: forge-gui/res/cardsfolder/upcoming/sibling_rivalry.txt [hunk 1]
```diff
@@ -0,0 +1,8 @@
+Name:Sibling Rivalry
+ManaCost:3 R
+Types:Sorcery
+A:SP$ GainControl | ValidTgts$ Creature | LoseControl$ EOT | Untap$ True | AddKWs$ Haste | SubAbility$ DBToken | SpellDescription$ Gain control of target artifact or creature until end of turn. Untap it. It gains haste until end of turn. Create a tapped Powerstone token. (It's an artifact with "{T}: Add {C}. This mana can't be spent to cast a nonartifact spell.")
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/sibling_rivalry.txt:4]: Wrong ValidTgts

File: forge-gui/res/cardsfolder/upcoming/skyfisher_spider.txt [hunk 2]
```diff
@@ -0,0 +1,15 @@
+Name:Skyfisher Spider
+ManaCost:2 B G
+Types:Creature Spider
+PT:3/3
+K:Reach
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigSac | TriggerDescription$ When CARDNAME enters the battlefield, you may sacrifice another creature. When you do, destroy target nonland permanent.
+SVar:TrigSac:AB$ ImmediateTrigger | Cost$ Sac<1/Creature.Other/another creature> | Execute$ TrigDestroy | TriggerDescription$ When you do, destroy target creature an opponent controls.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/skyfisher_spider.txt:7]: Wrong Description

File: forge-gui/res/cardsfolder/upcoming/skyfisher_spider.txt [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+SVar:TrigDestroy:DB$ Destroy | ValidTgts$ Permanent.nonLand | TgtPrompt$ Select target nonland permanent
+T:Mode$ ChangesZone | ValidCard$ Card.Self | Origin$ Battlefield | Destination$ Graveyard | Execute$ TrigExile | OptionalDecider$ TriggeredCardController | TriggerDescription$ When CARDNAME dies, you may gain 1 life for each creature card in your graveyard. If you do, exile CARDNAME from your graveyard.
+SVar:TrigExile:DB$ ChangeZone | Origin$ Graveyard | Destination$ Exile | Defined$ TriggeredNewCardLKICopy | SubAbility$ DBGainLife
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/skyfisher_spider.txt:10]: wrong effect order

File: forge-gui/res/cardsfolder/upcoming/slagstone_refinery.txt [hunk 4]
```diff
@@ -0,0 +1,8 @@
+Name:Slagstone Refinery
+ManaCost:4
+Types:Artifact 
+T:Mode$ ChangesZone | ValidCard$ Artifact.nonToken+YouCtrl+Other,Card.Self | Origin$ Battlefield | Destination$ Graveyard | Execute$ TrigToken | TriggerZones$ Battlefield | TriggerDescription$ Whenever CARDNAME or another nontoken artifact you control is put into a graveyard from the battlefield or is put into exile from the battlefield, create a tapped Powerstone token. (It's an artifact with "{T}: Add {C}. This mana can't be spent to cast a nonartifact spell.")
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/slagstone_refinery.txt:4]: Destination incomplete

File: forge-gui/res/cardsfolder/upcoming/hoarding_recluse.txt [hunk 5]
```diff
@@ -0,0 +1,10 @@
+Name:Hoarding Recluse
+ManaCost:3 G
+Types:Creature Spider
+PT:2/3
+K:Reach
+K:Deathtouch
+T:Mode$ ChangesZone | Origin$ Battlefield | Destination$ Graveyard | ValidCard$ Card.Self | Execute$ TrigChangeZone | TriggerDescription$ When CARDNAME dies, put up to one other target card from a graveyard on the bottom of its owner's library.
+SVar:TrigChangeZone:DB$ ChangeZone | Origin$ Graveyard | Destination$ Library | LibraryPosition$ -1 | ValidTgts$ Card
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/hoarding_recluse.txt:8]: Up to missing

File: forge-gui/res/cardsfolder/upcoming/third_path_iconoclast.txt [hunk 6]
```diff
@@ -0,0 +1,9 @@
+Name:Third Path Iconoclast
+ManaCost:U R
+Types:Creature Human Monk
+PT:2/1
+T:Mode$ SpellCast | ValidCard$ Card.nonCreature | ValidActivatingPlayer$ You | TriggerZones$ Battlefield | Execute$ TrigToken | TriggerDescription$ Whenever you cast a noncreature spell, create a 1/1 white Monk creature token with prowess.
+SVar:TrigToken:DB$ Token | TokenAmount$ 1 | TokenScript$ c_1_1_a_soldier
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/third_path_iconoclast.txt:6]: wrong token

File: forge-gui/res/cardsfolder/upcoming/third_path_iconoclast.txt [hunk 7]
```diff
@@ -0,0 +1,9 @@
+Name:Third Path Iconoclast
+ManaCost:U R
+Types:Creature Human Monk
+PT:2/1
+T:Mode$ SpellCast | ValidCard$ Card.nonCreature | ValidActivatingPlayer$ You | TriggerZones$ Battlefield | Execute$ TrigToken | TriggerDescription$ Whenever you cast a noncreature spell, create a 1/1 white Monk creature token with prowess.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/third_path_iconoclast.txt:5]: Wrong Description?

**Review by kevlahnota — APPROVED** [review]

**Pull request merged**